src/components/Layout/Header/NavBar/Notifications.tsx

```diff
@@ -40,14 +62,47 @@ export const Notifications = () => {
     }
   }, [colorMode, mobileBreakpoint])
 
-  if (!isWhereverEnabled || !currentWallet || !isKeyManagerWithProvider(currentWallet)) return null
+  if (
+    !isWhereverEnabled ||
+    !currentWallet ||
```

Comment:
we can remove this check, the `wallet` returned from `useWallet` hook is what you're after :)

Reply:
The problem with removing this check is that the condition below - `!eip712SupportedWallets.includes(currentWallet)` will error as it does not accept null, and then will need to be solved with in some other way, something like `!eip712SupportedWallets.includes(currentWallet || '' as KeyManager)`
WDYT?

Reply:
@nivo33 even better!